### PR TITLE
chore: Remove ppcbe Ubuntu

### PR DESCRIPTION
**Pull request by nschonni**
Only seems referenced by < Node 8

### CODE REVIEW

**Comment by sam-github** [comment]
I think ppcbe was last supported in 6.x, but not in 8.x and beyond, but those times predate me, it would be good to have a :+1: from @richardlau @BethGriggs or @mhdawson who were there at the time.

**Comment by richardlau** [comment]
As an aside it looks like `jenkins/xml/citgm-smoker.xml` is an old snapshot of the Jenkins config for `citgm-smoker` -- is there a policy of doing such snapshots for Jenkins configuration and keeping them up-to-date?

**Comment by sam-github** [comment]
Its my understanding they are snapshotted just before a job is deleted.

**Comment by nschonni** [comment]
@rvagg I think there are probably some manual steps when this lands given the feedback on some of the other cleanup PRs

**Comment by rvagg** [comment]
it shows up in these jobs on ci:

archive-libuv-test-pipeline
archive-node-test-known-issues
archive-node-test-npm
citgm-continuous-integration
citgm-smoker-nobuild
citgm-smoker-pipeline
llnode-continuous-integration
llnode-pipeline
node-inspect
nodereport-continuous-integration-latest
node-test-commit-plinux
node-test-node-addon-api
node-test-node-addon-api-new
readable-stream-continuous-integration
readable-stream-pipeline
string_decoder-continuous-integration
string_decoder-pipeline

and these on ci-release:

iojs+release
iojs+release-mdawson-major
iojs+release-mdawson-osx
iojs+release-mdawson-test-centosppc
iojs+release-mdawson-test
iojs+release-new-osx-temp
iojs+release-ppcle-only

if someone else has access to any of these it would be a help if you could hunt down where they are referenced (some in "labels" some in other places), otherwise I'll try and get to this sometime myself.

**Comment by richardlau** [comment]
> citgm-continuous-integration
> citgm-smoker-nobuild
> citgm-smoker-pipeline
> nodereport-continuous-integration-latest

I don't see any references to `ppcbe` in these jobs (at least in the web UI).

**Comment by rvagg** [comment]
@richardlau those 4 had it, ctrl-f for 'ppcbe' showed them mostly in some setup script:
```
  ppcbe-*) OS=linux; ARCH=ppc64; EXT=tar.gz;;
```
citgm-smoker-pipeline had it in the list of possible labels
removed all of those

still shows up in these in ci

archive-libuv-test-pipeline
archive-node-test-known-issues
archive-node-test-npm
llnode-continuous-integration
llnode-pipeline
node-inspect
node-test-commit-plinux
node-test-node-addon-api
node-test-node-addon-api-new
readable-stream-continuous-integration
readable-stream-pipeline
string_decoder-continuous-integration
string_decoder-pipeline

**Comment by sam-github** [comment]
Is this ready to land?

**Comment by mhdawson** [comment]
@sam-github Yes.  I had thought about landing it at one point but when I had looked at it I did not have time to stick around to validate that builds still worked afterwards. Since it affects the version selector script that is something I think is needed.

So just to say if you check back at a time where you can do that extra check go for it.

**Comment by nschonni** [comment]
Rebased this for the conflicts

File: jenkins/pipelines/llnode-pipeline.jenkinsfile [hunk 1]
```diff
@@ -7,7 +7,7 @@ properties([
     string(name: 'NODE_VERSIONS', defaultValue: 'v6 v8 v9 v10 nightly canary', description: 'Space separated list of NODE_VERSIONS to pass to llnode-continuous-integration.'),
     string(name: 'LLDB_VERSIONS', defaultValue: '3.9 4.0 5.0 6.0', description: 'Space separated list of LLDB_VERSIONS to pass to llnode-continuous-integration.'),
     string(name: 'MACHINES', defaultValue: 'all', description: '''Can be "all" or a space-separated list of nodes, eg:
-      debian8-64 fedora22 fedora23 osx1010 ppcle-ubuntu1404 ubuntu1204-64 ubuntu1404-64 ubuntu1604-64 rhel7-s390x aix61-ppc64 ppcbe-ubuntu1404 smartos16-64 smartos15-64 win10 win2012r2'''),
+      debian8-64 fedora22 fedora23 osx1010 ppcle-ubuntu1404 ubuntu1204-64 ubuntu1404-64 ubuntu1604-64 rhel72-s390x aix61-ppc64 smartos16-64 smartos15-64 win10 win2012r2'''),
```
Comment (AshCripps) [inline jenkins/pipelines/llnode-pipeline.jenkinsfile]: This should stay as `rhel7` as the `rhel72` machines are being turned off

Comment (nschonni) [reply jenkins/pipelines/llnode-pipeline.jenkinsfile]: Fixed. Guess this is a minor one since it's example text though

**Comment by sam-github** [comment]
@AshCripps Cleanup is good, could you land this, possibly doing any pre/post changes to make sure nothing is broken?

**Comment by AshCripps** [comment]
Im landing this now, went through the list of of jobs @rvagg found and removed any reference to `ppcbe-`. The only job I couldn't remove the reference was `node-test-commit-plinux`, this still has `ppcbe-ubuntu` as a configuration but I cant find any reference to it in the jobs config? Maybe theres something lying around on the jenkins master? 

Anyway, ill keep an eye for any failures but no jobs have been trying to use these machines for a long time so I dont see anything changing.

**Comment by sam-github** [comment]
The ppcbe ref in the plinux job must be historical, ppcbe support ended back in 6.x or 8.x, but the tag probably didn't get removed from the job.

**Comment by richardlau** [comment]
> 
> 
> The ppcbe ref in the plinux job must be historical, ppcbe support ended back in 6.x or 8.x, but the tag probably didn't get removed from the job.

https://ci.nodejs.org/job/node-test-commit-plinux is now not completing as it is waiting for `ppcbe-ubuntu1404` since `VersionSelectorScript.groovy` no longer excludes it from running.

Thing is, I can't see `ppcbe-ubuntu1404` in the web UI configure page for that job, only the `le` variant:
image

**Comment by AshCripps** [comment]
@nodejs/build-infra Could it be possible that theres some file/setting/etc on the main jenkins host that is where this job is pulling `ppcbe-ubuntu` from? is someone able to check?

**Comment by rvagg** [comment]
I think you probably removed the label (machines) before removing all the places it's used. If the label doesn't exist it doesn't show up in the list, but if it was ticked then it also doesn't show up in the list but remains ticked. A fun little bug.

Give another machine that label, then go through all the jobs where it's referenced and remove it, then unlabel the machine. Here's where it's referenced (when we get the backup repo set up you'll be able to grep this too):

```
/var/lib/jenkins/jobs# grep ppcbe-ubuntu1404 */config.xml
llnode-pipeline/config.xml:      debian8-64 fedora22 fedora23 osx1010 ppcle-ubuntu1404 ubuntu1404-64 ubuntu1604-64 rhel72-s390x aix61-ppc64 ppcbe-ubuntu1404 smartos16-64 smartos15-64 win10 win2012r2</description>
node-test-commit-plinux/config.xml:        <string>ppcbe-ubuntu1404</string>
node-test-node-addon-api/config.xml:ppcbe-ubuntu1404 smartos16-64 smartos15-64 win10 win2012r2 aix61-ppc64</description>
node-test-node-addon-api-new/config.xml:ubuntu1804-64 ppcbe-ubuntu1404 smartos16-64 smartos15-64 win10 win2012r2 aix61-ppc64 centos7-ppcle</description>
readable-stream-continuous-integration/config.xml:ppcbe-ubuntu1404 smartos16-64 smartos15-64 win10 win2012r2</description>
readable-stream-pipeline/config.xml:ppcbe-ubuntu1404 smartos16-64 smartos15-64 win10 win2012r2</description>
string_decoder-continuous-integration/config.xml:ppcbe-ubuntu1404 smartos16-64 smartos15-64 win10 win2012r2</description>
string_decoder-pipeline/config.xml:ppcbe-ubuntu1404 smartos16-64 smartos15-64 win10 win2012r2</description>
```

**Comment by AshCripps** [comment]
@rvagg Yeah the machines were already gone when I went to do this yesterday, I think theyve been gone a long time now. Ill add the label to a machine and go through and remove them now

**Comment by AshCripps** [comment]
@rvagg @richardlau Ive cleared it all up now, aborted any jobs stuck on `ppcbe-ubuntu1404` so this should be resolved now

**Pull request merged**